recipes/biogeme-optimization/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "biogeme-optimization" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme-optimization-{{ version }}.tar.gz
+  sha256: ad195d3ad6841d99ad8c53b118468ad7553b9a31d95d80b26614a87644c7d8d9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.24.2
+    - scipy >=1.10.1
+    - matplotlib-base >=3.8.2
+
+test:
+  imports:
+    - biogeme_optimization
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://biogeme.epfl.ch
+  summary: Various optimization algorithms for teaching and research
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
This is the error from the linux build:

```
ValueError: License file given in about/license_file (/home/conda/staged-recipes-copy/recipes/biogeme-optimization/LICENSE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

Reply:
There is no license file in the source tarball for this packaged. In fact there is no evidence of a license in the repo at all: https://github.com/michelbierlaire/optimization

If there is no license at all, I don't think we can add this to conda forge because no permission to redistribute is given. Maybe you can open an issue on the author's repository and ask if he will grant permission or add an OSS license to the repository.